EIPS/eip-3712.md

```diff
@@ -0,0 +1,321 @@
+---
+eip: 
+title: ERC Standard for Multiple Types of Fungible-Tokens
+author: Zhao Li(@1999321)、Yuefei Tan (@whtyfhas)、Derek Zhou (@zhous)  
+discussions-to:
```

Comment:
Will need to add a discussions link (usually on ethereum-magicians.org) before merge.